sunpy/net/vso/vso.py

```diff
@@ -705,13 +705,21 @@ def download(self, method, url, dw, callback, errback, *args):
 
     @staticmethod
     def by_provider(response):
+        """
```

Comment:
yes, I have sent a seperate PR for this. This PR is all jumbled up with other PRs, due to some patch mismatch. You can close this PR.